cranelift/filetests/filetests/runtests/atomic-load-store.clif

```diff
@@ -0,0 +1,105 @@
+test interpret
+test run
+set enable_llvm_abi_extensions=true
```

Comment:
There aren't any i128 tests here. Is the `enable_llvm_abi_extensions` flag necessary?